### PR TITLE
Add custom icons

### LINKED ISSUE #6: Add image grid view + better UI to Unit Converter
Add grid view to unit converter:
- [x] tapping on an item in the grid takes you to a new page (which will include the dropdowns for conversion)
- [x] Each grid item has an associated image

### PR DESCRIPTION
**Pull request by maryx**
Resolves #6 
Adds custom icons to the Unit Converter Category page.

### CODE REVIEW

File: unit_converter/unit_converter/lib/category_route.dart [hunk 1]
```diff
@@ -111,7 +111,7 @@ class _CategoryRouteState extends State<CategoryRoute> {
   }
 
   /// Retrieves a list of [Categories] and their [Unit]s
-  Future<Null> _retrieveLocalCategories() async {
+  Future<Null> _retrieveLocalCategories()   async {
```
Comment (yjbanov) [inline unit_converter/unit_converter/lib/category_route.dart]: nit: seems like an unintended change

File: unit_converter/unit_converter/lib/category_route.dart [hunk 2]
```diff
@@ -165,7 +165,7 @@ class _CategoryRouteState extends State<CategoryRoute> {
           name: apiCategory['name'],
           units: units,
           color: _baseColors[_baseColors.length - 1],
-          icon: _icons[_icons.length - 1],
+          iconLocation: _icons[_icons.length - 1],
```
Comment (yjbanov) [inline unit_converter/unit_converter/lib/category_route.dart]: hint: `_icons.last` would be more idiomatic (assuming `_icons` is some sort of `Iterable`, e.g. a `List`)

**Comment by yjbanov** [comment]
lgtm

**Comment by yjbanov** [comment]
Sorry for the delay.

File: unit_converter/unit_converter/lib/category_route.dart (hunk 2 above)
Comment (maryx) [reply unit_converter/unit_converter/lib/category_route.dart]: Thanks! Forgot `.last` exists!

File: unit_converter/unit_converter/lib/category_route.dart (hunk 1 above)
Comment (maryx) [reply unit_converter/unit_converter/lib/category_route.dart]: Thanks!

**Comment by maryx** [comment]
No problem, thanks for reviewing!

**Pull request merged**